Source/Plugin.BLE/Shared/ServiceBase.cs

```diff
@@ -61,16 +62,16 @@ public async Task<IReadOnlyList<ICharacteristic>> GetCharacteristicsAsync()
         /// Gets the first characteristic with the Id <paramref name="id"/>. 
         /// </summary>
         /// <param name="id">The id of the searched characteristic.</param>
-        public async Task<ICharacteristic> GetCharacteristicAsync(Guid id)
+        public async Task<ICharacteristic> GetCharacteristicAsync(Guid id, CancellationToken cancellationToken = default)
```

Comment:
Another warning because the new parameter is not documented:
```
Parameter 'cancellationToken' has no matching param tag in the XML comment for 'ServiceBase<TNativeService>.GetCharacteristicAsync(Guid, CancellationToken)' (but other parameters do)
```